### PR TITLE
Server side pagination bugs removed

### PR DESCRIPTION
**Pull request by Josephalexantony-aot**

### CODE REVIEW

File: forms-flow-web/src/components/Dashboard/Dashboard.js [hunk 1]
```diff
@@ -378,10 +357,11 @@ const Dashboard = React.memo(() => {
                     className="form-select mx-5 mb-3"
                     aria-label="Choose page limit"
                   >
-                    <option selected>6</option>
-                    <option value={12}>12</option>
-                    <option value={30}>30</option>
-                    <option value={9000}>All</option>
+                    <option value={limit}>{limit}</option>
```
Comment (abhilash-aot) [inline forms-flow-web/src/components/Dashboard/Dashboard.js:360]: Use an array with the numbers and iterate options

File: forms-flow-web/src/components/Dashboard/Dashboard.js [hunk 2]
```diff
@@ -378,10 +357,11 @@ const Dashboard = React.memo(() => {
...
+                    {limit != 6 && <option value={6}>6</option>}
+                    {limit != 12 && <option value={12}>12</option>}
+                    {limit != 30 && <option value={30}>30</option>}
+                    {limit != 9000 && <option value={9000}>All</option>}
```
Comment (abhilash-aot) [inline forms-flow-web/src/components/Dashboard/Dashboard.js:364]: Why is limit 9000??

Comment (abhilash-aot) [reply forms-flow-web/src/components/Dashboard/Dashboard.js:364]: That should be dynamic

**Review by abhilash-aot — CHANGES_REQUESTED** [review]

File: forms-flow-web/src/components/Dashboard/Dashboard.js (hunk 2 above)
Comment (Josephalexantony-aot) [reply forms-flow-web/src/components/Dashboard/Dashboard.js:364]: sure

**Review by seethalelias-aot — APPROVED** [review]

**Review by Josephalexantony-aot — COMMENTED** [review]
Made changes as comments

File: forms-flow-web/src/components/Dashboard/Dashboard.js (hunk 2 above)
Comment (Josephalexantony-aot) [reply forms-flow-web/src/components/Dashboard/Dashboard.js]: done

**Review by AbijithS-aot — APPROVED** [review]

**Review by seethalelias-aot — APPROVED** [review]

**Pull request merged**